.prettierrc

```diff
@@ -1,4 +1,9 @@
 {
+    "trailingComma": "es5",
```

Comment:
@devmailar You can see that your commit got messed up :8). Try fixing it and we'll merge.

Reply:
Ok I fixed it